app/views/club_applications/new.html.haml

```diff
@@ -24,8 +24,10 @@
         .small-6.columns
           = f.input :last_name, placeholder: 'Beeblebrox'
       .row
-        .columns
+        .small-6.columns
           = f.input :email, placeholder: 'zaphod@beeblebrox.com'
+        .small-6.columns
+          = f.input :phone_number, label: "Phone number (we'll get in touch to answer any questions you might have)",placeholder: '(555) 555-5555'
```

Comment:
`you might have)",placeholder: '(555) 555-5555'`

should be

`you might have)", placeholder: '(555) 555-5555'` (extra space after comma)